genai/example_test.go

```diff
@@ -101,6 +101,29 @@ func ExampleGenerativeModel_GenerateContent_safetySetting() {
 	printResponse(resp)
 }
 
+func ExampleGenerativeModel_GenerateContent_codeExecution() {
+	ctx := context.Background()
+	client, err := genai.NewClient(ctx, option.WithAPIKey(os.Getenv("GEMINI_API_KEY")))
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer client.Close()
+
+	model := client.GenerativeModel("gemini-1.0-pro")
```

Comment:
Nit: let's use newer models for new examples; I think there was a request to suggest using new models in docs/examples whenever possible
you're using 1.5-flash for the test...

Reply:
Done here and elsewhere in the examples.